phpcs.xml

```diff
@@ -1,14 +1,18 @@
 <?xml version="1.0"?>
 <ruleset name="Webimpress Coding Standard">
-    <rule ref="./vendor/webimpress/coding-standard/ruleset.xml"/>
+    <rule ref="./vendor/webimpress/coding-standard/ruleset.xml">
```

Comment:
We should probably move to Laminas Coding Standard v2, but that can wait for another PR. :smile: